src/todos/schemas/todos.schema.ts

```diff
@@ -12,6 +12,19 @@ export class Todo {
 
   @Prop()
   description?: string
+
+  // ? Question: is it a correct way to attach refs to the models?
```

Comment:
If you're about to use `mongoose`' `populate()` — yes.
https://mongoosejs.com/docs/populate.html

Otherwise it's not required, from MongoDB perspective it's just as usual field as any other.